samples/skills/WriterSkill/TranslateV2/config.json

```diff
@@ -3,7 +3,7 @@
   "type": "completion",
   "description": "Translate text to English and improve it",
   "completion": {
-    "max_tokens": 4000,
+    "max_tokens": 3000,
```

Comment:
We should be careful about needing to modify token limits of skills just to get it working with the Planner. Especially because I don't expect this to be used by the examples currently (it needs some input descriptions).

Reply:
@lemillermicrosoft Modified those to pass the GPT3 integration and plan examples. Was getting error due to not enough tokens for input + completion